dataprofiler/data_readers/data_utils.py

```diff
@@ -163,7 +187,7 @@ def read_json_df(data_generator, selected_columns=None, read_in_string=False):
         each call as well as original dtypes of the dataframe columns.
     :rtype: typle(Iterator(pd.DataFrame), pd.Series(dtypes)
     """
-    lines = list()
+    lines: List[Dict] = list()
```

Comment:
not necessarily the case... could be a list of `str` / `list` / `'dict`, @Sanketh7?

Reply:
This relates to the comment above about it being JSON

Reply:
Fixed to accept all json types.